javascript/javascript1/week4/getReply.js

```diff
@@ -0,0 +1,103 @@
+let userName = " ";
+let todo = [];
+function getReply(command) {
+  //'ask for name.'
+  const array = command.split(" ");
+  let response = " ";
+  if (command.includes("hello my name is ")) {
+    userName = array[array.length - 1];
+    return "Nice to meet you " + userName;
+  } else if (command.includes("what is my name")) {
+    return "your name is " + userName;
+  } else if (command.includes() {
+    return "your name is not find";
+  }
+  //'adding to todos.'
+  else if (command.includes("add ") && command.includes(" to my todos")) {
+    todo.push("fishing");
+    todo = command(command.length - 1);
+    return "fishing is added to your todos";
+  } else if (command.includes("add signging")) {
+    todo.push("signging");
+  } else if (command.includes("remove fishing")) {
+    for (let i = 0; i < todo.length; i++) {
+      if (todo[i].includes("fishing")) {
+        todo.splice(i, 1);
+        return "removed fishing from your todo";
+      }
+    }
+  }
+  else if(command.includes("on my todo"))
+    {
+        return(todo);
+    }
+  else if (command.includes("is it today?")) {
+    const monthNames = [
+      "January",
+      "February",
+      "March",
+      "April",
+      "May",
+      "June",
+      "July",
+      "August",
+      "September",
+      "October",
+      "November",
+      "December",
+    ];
+    const today = new Date();
+    const dd = String(today.getDate()).padStart(2, "0");
+    const yyyy = today.getFullYear();
+    console.log(`${dd}. of ${monthNames[today.getMonth()]} ${yyyy}`);
```

Comment:
This looks pretty nice. Only you need to return instead of console.log. Because on line 98 you are already logging out the returned value of the getReply function.